internal/tracing/manifests/otelcol/config.go

```diff
@@ -0,0 +1,15 @@
+package otelcol
+
+import (
+	"github.com/ViaQ/logerr/v2/kverrors"
+	"gopkg.in/yaml.v2"
+)
+
+func ConfigFromString(configStr string) (map[string]interface{}, error) {
```

Comment:
Does it make sense to reuse the parsing functionality introduced in the operator api `v2alpha2`?

Jacob is currently working on extending the parsing functionality. Which means in the future we will get port parsing and other stuff well tested for a low effort.